### PR TITLE
Refactor the RPC endpoints and providers page

### PR DESCRIPTION
**Pull request by mahsamoosavi**
- Fixes the broken links within the node providers table.
- Removes the `Arb Goerli ` column from the node providers table.
- Adds Websocket support to the node providers table.
- Adds a note to the RPC providers page about the WebSocket calls not being supported when using the Arbitrum endpoints.

[Preview](https://nitro-docs-6opjdrzn8-offchain-labs.vercel.app/node-running/node-providers#rpc-providers)

### CODE REVIEW

File: arbitrum-docs/node-running/node-providers.mdx [hunk 1]
```diff
@@ -26,6 +26,8 @@ This section provides an overview of the available public RPC endpoints for diff
 
 ⚠️ Unlike the RPC Urls, the Sequencer endpoints only support `eth_sendRawTransaction` and `eth_sendRawTransactionConditional` calls.
 
+⚠️ Important note: WebSocket calls are not supported when using the Arbitrum public endpoints.
```
Comment (TucksonDev) [inline arbitrum-docs/node-running/node-providers.mdx:29]: I'd say something like "Arbitrum public RPCs do not provide websocket support" to make it more clear.

File: arbitrum-docs/node-running/node-providers.mdx [hunk 2]
```diff
@@ -26,6 +26,8 @@ This section provides an overview of the available public RPC endpoints for diff
...
+
 ⚠️ Important note: Ethereum Goerli was deprecated in Q1 2023 but will be supported until Q4 2023. Once Arbitrum Sepolia becomes publicly available, we will phase out the usage of Arbitrum Goerli. Support for Arbitrum Goerli will cease prior to the conclusion of Ethereum Goerli.
```
Comment (TucksonDev) [inline arbitrum-docs/node-running/node-providers.mdx:31]:
```suggestion
⚠️ Important note: Ethereum Goerli was deprecated in Q1 2023 but will be supported until Q1 2024. Once Arbitrum Sepolia becomes publicly available, we will phase out the usage of Arbitrum Goerli. Support for Arbitrum Goerli will cease prior to the conclusion of Ethereum Goerli.
```
I think? Not 100% sure.

Comment (mahsamoosavi) [reply arbitrum-docs/node-running/node-providers.mdx:31]: I think that's correct (based on a little search). But let's see if @amarrazza can confirm.

Comment (mahsamoosavi) [reply arbitrum-docs/node-running/node-providers.mdx:31]: @TucksonDev I'll commit your suggestion and resolve the convo as I'm 98% sure it's correct

**Review by TucksonDev — APPROVED** [review]

**Pull request merged**